metron-analytics/metron-maas-service/README.md

```diff
@@ -138,7 +138,7 @@ Now that we have a deployed model, let's adjust the configurations for the Squid
 * Edit the squid parser configuration at `$METRON_HOME/config/zookeeper/parsers/squid.json` in your favorite text editor and add a new FieldTransformation to indicate a threat alert based on the model (note the addition of `is_malicious` and `is_alert`):
 ```
 {
-  "parserClassName": "org.apache.metron.parsers.GrokParser",
+  "parserClassName": "org.apache.metron.parsers.grok.GrokParser",
```

Comment:
Yes that is my thinking.  They, along with JSONMap provide base capabilities to be extended by other parsers, or to be used with configurations to make they themselves useful.  So if we added an XML or scripting parser, it would go in here as well.

The extensions - in the context of these types - are the bundling of specific configurations of these base parsers ( indexing, parser, enrichment, ES ) into a single installable unit.
You could also do this manually by pushing configurations to the right places in zookeeper and changing ambari etc, and you can still do this in a post 777 world.
Extensions also give a project structure for implementing and executing unit and integration tests, on even configuration only parsers ( SEE: YAF, SQUID ).  This I think should be the recommended way, even the supported way for production.  
Along with METRON-942 that is.